src/main.c

```diff
@@ -216,12 +216,14 @@ static int process(jq_state *jq, jv value, int flags, int dumpopts) {
     jv_free(exit_code);
     jv error_message = jq_get_error_message(jq);
     if (jv_get_kind(error_message) == JV_KIND_STRING) {
-      fprintf(stderr, "jq: error: %s", jv_string_value(error_message));
+      // No prefix should be added to the output of `halt_error`.
+      fwrite(jv_string_value(error_message), 1,
+          jv_string_length_bytes(jv_copy(error_message)), stderr);
```

Comment:
I guess we don't leak the `jv` copy here as `jv_string_length_bytes` free/unref? I will have to read up on how the ownership works for the `jv` type and functions, eg why does `jv_string_value` not free/unref?